src/main/java/com/foundrysoftware/minecraft/plugins/PluginTemplate.java

```diff
@@ -87,12 +87,12 @@ public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command
                                 if(item == null){
                                     continue;
                                 }
-                                if(item.getType()== Material.TIPPED_ARROW|| item.getType() == Material.ARROW || item.getType()==Material.SPECTRAL_ARROW){
+                                if(item.getType().toString().contains("ARROW")){
                                     item.setType(Material.TIPPED_ARROW);
                                     item.setAmount(item.getAmount());
                                     PotionMeta potionmeta = (PotionMeta)item.getItemMeta();
                                     potionmeta.setBasePotionData(new PotionData(PotionType.getByEffect(PotionEffectType.getByName(args[0]))));
-                                    potionmeta.addCustomEffect(new PotionEffect(PotionEffectType.getByName(args[0]),10,10),true);
+                                    potionmeta.addCustomEffect(new PotionEffect(PotionEffectType.getByName(args[0]),10,10),true);//duration,amplifire,Show particles
```

Comment:
please create constants/variable with the names from the comments and use those as the parameters

Reply:
wont that cause more lag?

Reply:
done